src/jit/lower.cpp

```diff
@@ -2862,8 +2862,8 @@ void Lowering::InsertPInvokeMethodProlog()
     firstBlockRange.InsertBefore(insertionPoint, LIR::SeqTree(comp, store));
     DISPTREERANGE(firstBlockRange, store);
 
-#ifndef _TARGET_X86_ // For x86, this step is done at the call site (due to stack pointer not being static in the
-                     // function).
+#if !defined(_TARGET_X86_) && !defined(_TARGET_ARM_)
+    // For x86 and arm32, this step is done at the call site (due to stack pointer not being static in the function).
 
```

Comment:
This doesn't apply to arm32, since arm32 has a static SP throughout the function. Thus, offsetOfCallSiteSP for arm32 should be handled the same as amd64 (set in function prolog, not set at pinvoke call site).